decidim-core/db/migrate/20220823094517_add_time_zone_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTimeZoneToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :decidim_users, :time_zone, :string, limit: 255, default: "UTC"
```

Comment:
```suggestion
    add_column :decidim_users, :time_zone, :string
```
Let's not add a default here, we will use the organization timezone if this is empty